OracleEDQ/12.2.1.3.0/EDQServer.env.list

```diff
@@ -0,0 +1,2 @@
+ADMIN_HOST=<EDQ Admin Server Host>
+ADMIN_PORT=<EDQ Admin Server Port>
```

Comment:
Can you not provide the default port?

Reply:
File EDQServer.env.list removed. As it is replaced by command line paramerters.